tests/format/MemberAccess/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -195,6 +248,59 @@ contract MemberAccess {
                 path,
                 address(this, aoeu, aoeueu, aoeu)
             );
+
+        // Comment before chain
+        // CONFIG
+        // Comment 1
+        // Comment 2
+        // Comment 3
+        // Comment 4
+        // Comment 5
```

Comment:
Is this what we would like to achieve? Move all the comments above the chain?

Reply:
Yes
Have a look at #644 and also try it on Prettier JS to see what they are doing at the moment.
It's not perfect but there is something nice in it.

Reply:
I'll give it a try